.github/workflows/ci.yaml

```diff
@@ -0,0 +1,45 @@
+---
+
+name: Continuous integration
+
+on:
+  push:
+
+jobs:
+  main:
+    runs-on: ubuntu-18.04
+    name: Continuous integration
+    timeout-minutes: 60
```

Comment:
One hour? Isn't that a bit exessive?

Reply:
Copy past from bigger project :-), thanks :-)